VK-Clone/VK-Clone/Services/VKAPIService.swift

```diff
@@ -98,13 +132,14 @@ final class VKAPIService {
     }
 
     func getGroupsOfVKUser() {
```

Comment:
```suggestion
    func getGroupsVKUser() {
```
of по гайду не пишем